core/src/idx/planner/mod.rs

```diff
@@ -120,13 +123,13 @@ impl<'a> QueryPlanner<'a> {
 	fn add(
 		&mut self,
 		tb: Table,
-		irf: Option<IteratorRef>,
+		ir: Option<IteratorRef>,
```

Comment:
This doesn't need to be addressed in this PR, but it would be good to have consistent naming, either `it_ref` (which I liked), `irf` or `ir`. Don't mind which, but it_ref was the most obvious without looking at the types

Reply:
Well spotted! It should be `irf` indeed! Fixed now!